storage/service/src/reader_threads.rs

```diff
@@ -31,6 +28,8 @@ pub fn init_thread_pool(reader_threads: ReaderThreads) -> Arc<ThreadPool> {
 }
 
 //---------------------------------------------------------------------------------------------------- ReaderThreads
+/// Amount of database reader threads to spawn
+///
 /// This controls how many reader thread `service`'s
 /// thread-pool will spawn to receive and send requests/responses.
```

Comment:
> Ok, it does feel a bit redundant though as the next sentence says pretty much the same thing.

The purpose of the first line is to be a brief description for `cargo doc`.

![image](https://github.com/user-attachments/assets/baf3fc8c-1623-4b29-9b4f-3df2d6426231)

> The following documentation mentioning a service module must be updated too.

This wasn't updated.

```suggestion
/// Amount of database reader threads to spawn.
///
/// This controls how many reader threads the [`DatabaseReadService`](crate::DatabaseReadService)
/// thread-pool will spawn to receive and send requests/responses.
```